aiida_common_workflows/workflows/relax_and_bands.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+"""
+Workflow that runs a relaxation and subsequently calculates bands.
+It can use any code plugin implementing the common relax workflow and the
+common bands workflow.
+It also allows the automatic use of `seekpath` in order to get the high
+symmetries path for bands.
+"""
```

Comment:
As a general comment, please follow the standard docstring guidelines.

First line should be on the same line as the opening quotes and should not span more than one line.
Then there should be a empty line, followed by the rest. The width should fill the 120 characters just like the code and not add premature linebreaks.

As an example, this docstring would become:
```
"""Workflow that runs a relaxation and subsequently calculates bands.

It can use any code plugin implementing the common relax workflow and the common bands workflow. It also allows the
automatic use of ``seekpath`` in order to get the high symmetries path for bands.
"""

Note that to have monospace font in the docs, you need double backticks (since it is RST and not markdown).